### PR TITLE
UF-8938: Add municipalityid (API major)

### PR DESCRIPTION
**Pull request by ronnberg**

### CODE REVIEW

File: src/main/java/se/sundsvall/templating/integration/db/entity/TemplateEntity.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
     name = "templates",
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/templating/integration/db/entity/TemplateEntity.java:40]: municipality_id is included in the query conditions. Add index on column?

File: src/main/java/se/sundsvall/templating/configuration/MunicipalityIdConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.context.annotation.RequestScope;
+import se.sundsvall.templating.domain.ContextMunicipalityId;
+
+@Configuration
+public class MunicipalityIdConfiguration {
+
+	@Bean
+	@RequestScope
+	public ContextMunicipalityId requestScopedBean() {
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/templating/configuration/MunicipalityIdConfiguration.java:13]: Remove public keyword?

File: src/main/java/se/sundsvall/templating/domain/ContextMunicipalityId.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package se.sundsvall.templating.domain;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ContextMunicipalityId {
+	private String id;
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/templating/domain/ContextMunicipalityId.java:9]: The class is named ContextMunicipalityId and wraps a municipalityId.

Rename `id `to `value`?

File: src/main/java/se/sundsvall/templating/integration/db/TemplateRepository.java [hunk 4]
```diff
@@ -16,52 +17,69 @@
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/templating/integration/db/TemplateRepository.java:17]: Add circuitBreaker?

**Comment by foobar-1337** [comment]
Fix sonar issues

**Review by jklingst — APPROVED** [review]

**Pull request merged**